content/en/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -51,6 +51,8 @@ Unlike [names and UIDs](/docs/user-guide/identifiers), labels do not provide uni
 
 Via a _label selector_, the client/user can identify a set of objects. The label selector is the core grouping primitive in Kubernetes.
 
+When a Kubernetes object contains a selector, the selector matches objects only in the same namespace as the object containing the selector.
+
```

Comment:
it is inaccurate. in many cases, it is right, but as for volume objects, such as PVC/PV, their relations are not limited in one namespaces.
/cc @tengqm

Reply:
Yup. Not all resources are namespaced.

Reply:
I see. Thanks @zhangxiaoyu-zidif and @tengqm. I can see that the general statement I made is inaccurate.

However, is there a more specific clarification that can be made? You see I'm still unsure whether the selector of a service defined in namespace "foo" could match a pod in namespace "bar" with labels matching the service's selector. Without a clarification, I'll need to add something like a "namespace" label to avoid accidentally  matching pods across namespaces. Or maybe there's a better place to clarify this behaviour in the docs. Please advise.

(I'm trying to depend on documented behaviour rather than simply on how kubernetes happens to work at this point in time.)

Reply:
@tenqm @zhangxiaoyu-zidif 👋 Any advice for how @glyn can follow up?